code/__HELPERS/traits.dm

```diff
@@ -28,3 +28,21 @@
 	else if(isatom(target))
 		var/atom/the_atom2 = target
 		REMOVE_TRAIT(the_atom2,trait,source)
+
+/// Define wrapper for adding multiple traits to a target via a list. Must have a common source and target.
+#define ADD_TRAITS(target, list_of_traits, source) _add_traits(target, list_of_traits, source)
```

Comment:
There shouldn't be any macro, there's no point!

Reply:
Ideal interface is `add_traits(target, list_of_traits, source)`. There is no need for the indirection

Reply:
keeping it as a macro helps out with linting in case you fumble the arguments somehow (i.e. forgetting a list, implementing multiple sources), which isn't something i believe you can do with procs? i can remove it too but i figured that was neat enough for me to keep the macro

Reply:
I don't like that logic because nearly all proc parameters are required

Reply:
Ideally we'd have a lint but our tools are not good enough for that (please do not fucking regex this)

Reply:
i also don't like the potential syntax of people trying to `target.add_traits(...)` in the thought that that might do something, very unlikely but it just feels like something that can be confused like that

Reply:
I think it *should* be target.add_traits tbh